### ISSUE TITLE
Documentation?

### ISSUE THREAD

**Issue by willhaslett** [google/charts#440]
Hi. This is a great package and a deal-sealer for our group. But, where do I go for actual API documentation? Currently, I'm grepping the source to figure out what various classes want to be fed. Is there a better way?

**Comment by pluus** [comment]
+1

**Comment by rakakhrl** [comment]
I agree, from what i can see this package tick all the boxes in term of feature but sadly there's no proper documentation

**Comment by peterlauri** [comment]
The examples are pretty good and comprehensive in case you haven't already looked. Examples are located here: https://github.com/google/charts/tree/master/charts_flutter/example

**Comment by willhaslett** [comment]
Examples are great. They don't replace API documentation. Want to know what a particular class can do in this package? Well, the examples show you one snapshot of that space.

**Comment by peterlauri** [comment]
We are all different. I prefer a rich set of examples, rather than
documentation :)

**Comment by willhaslett** [comment]
I'm using `SlidingViewportOnSelection`. It auto-scales the Y based on what's visible at any moment, rather than keeping a fixed y-axis scale based on all of the loaded data. Is this configurable? Is there an optional argument that can be passed?

There are innumerable cases like this, and that's why examples, however helpful, are not a replacement for documentation. If you want to know what the classes can do that's not in the examples, you have to start reading source.

**Comment by palicka** [comment]
+1
